include/crow/middlewares/cookie_parser.h

```diff
@@ -140,6 +142,16 @@ namespace crow
         private:
             Cookie() = default;
 
+            static void dumpString(std::stringstream& ss, bool cond, const char* prefix,
+                                   boost::optional<const std::string&> value = boost::none)
```

Comment:
Would it be possible to just set the `value` parameter as an optional `const std::string&` and passing an empty string as a default value?

The reason I'm saying this is because I'm trying to reduce the reliance on boost as much as possible so that removing it as a dependency later is made easier.